github_stargazers/github_stargazers.py

```diff
@@ -7,53 +7,37 @@
 from github_stargazers.github import UsernameRepositoryError, TooManyRequestsHttpError, HTTPError
 
 
+class OutputPrintable(object):
```

Comment:
I think that it might be better to send a separate PR only with this class and its usages and then another PR with the changes from `_Command`.